apps/prairielearn/src/models/assessment.sql

```
@@ -154,3 +154,29 @@ FROM
+FROM
+  zones AS z
+  LEFT JOIN LATERAL (
+    SELECT
+      coalesce(sum(aq.max_points), 0) AS question_points
+    FROM
+      alternative_groups AS ag
+      JOIN assessment_questions AS aq ON (aq.alternative_group_id = ag.id)
+    WHERE
+      ag.zone_id = z.id
+      AND aq.deleted_at IS NULL
+  ) AS zq ON true
```

SQLFluff (4.1.0)

[error] Boolean/null literals must be consistently upper case.

(CP04)